### PR TITLE
Rework Features Page

### PR DESCRIPTION
**Pull request by brandonhundt**
- Fleshes out the features page with more content. 
- Once merged this format will make it easier to add more cards when features are rolled out. Possibly linking to a blog post.

## To review
- [ ] `bundle exec jekyll server`
- [ ] go to [http://localhost:4000/features](http://localhost:4000/features)
- [ ] Wordsmithing appreciated. Ensure new layout looks okay.

Here's a screenshot:
localhost_4000_features

### CODE REVIEW

File: pages/features.md [hunk 1]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+            <li class="list-group-item">Analyze across geography, device, and podcast app</li>
+          </ul>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          {% picture img/features/ulm.png 16:9 class="card-img-top" --alt Screenshot of Unique Listener Metrics dashboard %}
+          <div class="card-body">
+            <h3>Unique Listener Metrics</h3>
+            <p class="mb-0">Understand how your listener base is growing or shrinking over time.</p>
```
Comment (cavis) [inline pages/features.md:127]: "Growing or shrinking" I don't like... maybe "Understand how your listener base changes over time"?

File: pages/features.md [hunk 2]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+          </div>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">IAB 2.1 Compliant</li>
+            <li class="list-group-item">Filter by date</li>
+            <li class="list-group-item">Easily accessed via dashboard</li>
```
Comment (cavis) [inline pages/features.md:132]: "Filter by Date" and "Easily accessed" feel non-descriptive.  Maybe something about the rolling-vs-calendar rollups?  And/or the 7 vs 30 day windows?

File: pages/features.md [hunk 3]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+        </div>
+      </div>
+    </div>
   </div>
 </section>
 
-<section class="bg-gray-x p-5">
+<section class="bg-white p-5">
   <div class="container col-xxl-8">
     <h2 class="display-6 mb-3">Sell with confidence using Ad Inventory Management</h2>
```
Comment (cavis) [inline pages/features.md:142]: I almost want to throw "_Predictive_ Inventory Management" in here...

File: pages/features.md [hunk 4]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
-    <p>Dovetail automatically predicts and adjusts ad distribution for every campaign daily to ensure your orders are completed on time.</p>
-    <p><a href="/2022/07/21/introducing-dovetail-as-product">Learn more about inventory management</a></p>
+    <p>Understand your inventory with best in class inventory management. <a href="/2022/07/21/introducing-dovetail-as-product">Learn more in our blog post</a>.</p>
+    <div class="row row-cols-1 row-cols-md-2 g-4">
+      <div class="col">
+        <div class="card">
+          {% picture img/features/forecast.png 16:9 class="card-img-top" --alt screenshot of a forecast in Dovetail %}
+          <div class="card-body">
+            <h3>Forecasting</h3>
+            <p class="mb-0">Our download forecasts are integral to how campaigns are served.</p>
```
Comment (cavis) [inline pages/features.md:150]: "An accurate download forecast is integral to fulfilling your sales goals"?  Or "campaign goals"?

File: pages/features.md [hunk 5]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+        <div class="card">
+          {% picture img/features/campaign-management.png 16:9 class="card-img-top" --alt screenshot of campaign and flight entry in Dovetail %}
+          <div class="card-body">
+            <h3>Campaign Management</h3>
+            <p class="mb-0">Quickly add new IOs and segment them as they need.</p>
+          </div>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">Flight ads by targets</li>
+            <li class="list-group-item">Modify priorities</li>
+            <li class="list-group-item">Near real time forecasts as you input</li>
```
Comment (cavis) [inline pages/features.md:169]: I'd say "availability" here (or "allocations", if you want to define that).  Not "Forecasts".

File: pages/features.md [hunk 6]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+            <li class="list-group-item">Adjusts daily</li>
+          </ul>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          {% picture img/features/campaign-management.png 16:9 class="card-img-top" --alt screenshot of campaign and flight entry in Dovetail %}
+          <div class="card-body">
+            <h3>Campaign Management</h3>
+            <p class="mb-0">Quickly add new IOs and segment them as they need.</p>
```
Comment (cavis) [inline pages/features.md:164]: "Segment them as needed"?

File: pages/features.md [hunk 7]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+            <li class="list-group-item">Near real time forecasts as you input</li>
+          </ul>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          {% picture img/features/reporting.png 16:9 class="card-img-top" --alt screenshot of availability search in Dovetail %}
+          <div class="card-body">
+            <h3>Reporting</h3>
+            <p class="mb-0">Look forward and backwards to produce reports</p>
```
Comment (cavis) [inline pages/features.md:178]: "Reports on forecasted inventory as well as historical performance"?

File: pages/features.md [hunk 8]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+          </div>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">Search availability across shows</li>
+            <li class="list-group-item">Build your own campaign reports</li>
+            <li class="list-group-item">QA checks so you can have confidence the right ad is going to be served</li>
```
Comment (cavis) [inline pages/features.md:183]: Backsaw!

(I almost wish the "schedule" had its own card.  Maybe someday).

I might describe it a bit differently though.  "Simulate 100s download requests, to ensure you know what combinations of ads will be served on an episode".

File: pages/features.md [hunk 9]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+          </ul>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          {% picture img/features/programmatic.png 16:9 class="card-img-top" --alt Screenshot of category blockers in Dovetail %}
+          <div class="card-body">
+            <h3>Privacy forward Programmatic</h3>
+            <p class="mb-0">Backfill ad inventory by hooking into privacy forward ad-exchanges</p>
```
Comment (cavis) [inline pages/features.md:192]: Backfill?  Maybe just "fill remnant ad inventory"?  (I kind of want the word "remnant" somewhere in here).

File: pages/features.md [hunk 10]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+          </div>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">VAST tag support in creative library</li>
+            <li class="list-group-item">Brand Safety</li>
+            <li class="list-group-item">Opt-outs</li>
```
Comment (cavis) [inline pages/features.md:197]: Say more?

**Review by cavis — APPROVED** [review]
Some suggestions!

(Generally, this looks good to me though.  So feel free to keep wordsmithing, or just ship it!)

File: pages/features.md [hunk 11]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
-    <div>{% picture img/hero-multiple-feeds.jpg --alt woman at laptop wearing headphones %}</div>
-  </div>
-  <div class="pt-5 pb-4">
-    <div class="hero-content container col-xxl-8">
-      <div class="row">
-        <div class="icon-svg quote-mark d-flex  justify-content-center col-5">
-          <img src="/assets/img/rss-outline.svg" alt="RSS Icon" aria-hidden="true" class="" width="100" height="100" />
+<section class="bg-white p-5">
+  <div class="container col-xxl-8">
+    <h2 class="display-6 mb-3">Content and Feed Management with ease</h2>
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply pages/features.md:40]: This was marketing language that I've kept. Yes, theoretically its the same thing, but spelling it out is helpful.

File: pages/features.md [hunk 12]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+    <p class="lead">Producer tools for episode uploads and feed management</p>
+    <div class="row row-cols-1 row-cols-md-2 g-4">
+      <div class="col">
+        <div class="card">
+          {% picture img/features/multiple-feeds.png 16:9 class="card-img-top" --alt Screenshot of Multiple Feeds form %}
+          <div class="card-body">
+            <h3>Multiple Feeds</h3>
+            <p class="mb-0">We support multiple business models</p>
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply pages/features.md:48]: Agreed. Modifying slightly.

File: pages/features.md [hunk 13]
```diff
@@ -35,40 +35,170 @@ image: /assets/img/og-image.jpg
...
+          </div>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">Private Ad-free feeds</li>
+            <li class="list-group-item">Bonus Content</li>
+            <li class="list-group-item">Subscription service support</li>
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply pages/features.md:53]: I'm very comfortable making this claim.

File: pages/features.md (hunk 2 above)
Comment (brandonhundt) [reply pages/features.md:132]: Thanks!

File: pages/features.md (hunk 10 above)
Comment (brandonhundt) [reply pages/features.md:197]: This is one, we don't have yet...

`Allow listeners to Opt-out of participating in privacy exchanges`

Comment (brandonhundt) [reply pages/features.md]: I commented this out for now, until we get the opt-out working. Privacy is a feature!

**Pull request merged**